test/unit/helpers/orderBy.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+var chai = require('chai');
+var expect = require('chai').expect;
+var OrderBy = require('../../../helpers/orderBy');
+
+describe('OrderBy', function () {
+
+	var sortFields = ['afield'];
+
+	describe('when orderBy = null', function () {
+		var orderBy = OrderBy(null, {sortFields: sortFields});
+		it('should return sortField = null', function () {
+			expect(orderBy.sortField).to.be.null;
+		});
+		it('should return sortMethod = null', function () {
+			expect(orderBy.sortMethod).to.be.null;
+		});
+		it('should return error = undefined', function () {
+			expect(orderBy.error).to.be.undefined;
+		});
+	});
+
+	describe('when desc is in lowercase', function () {
+		it('converts desc to uppercase', function () {
+			var orderBy = OrderBy('afield:desc', {sortFields: sortFields});
+			expect(orderBy.sortMethod).to.equal('DESC');
+		});
+	});
+
+	describe('when asc is in lowercase', function () {
+		it('converts asc to uppercase', function () {
+			var orderBy = OrderBy('afield:asc',{sortFields: sortFields});
+			expect(orderBy.sortMethod).to.equal('ASC');
+		});
+	});
+
+	describe('quoteField', function () {
+		it('defaults to true', function () {
```

Comment:
We are not testing here if it use `true`, but testing results, so:
`should add double quotes (default) when omitted`